Cargo.toml

```
@@ -23,6 +23,7 @@ spec = "internal/config_specification.toml"
 [dependencies]
 anyhow = "1.0"
 bitcoin = { version = "0.30.0", features = ["serde", "rand-std"] }
+bitcoin_slices = { version = "0.6", features =["bitcoin","sha2"] }
```

nit :slightly_smiling_face: 

```suggestion
bitcoin_slices = { version = "0.6", features =["bitcoin", "sha2"] }
```

---

Good catch, thanks!
Will be fixed by 2e6191e229def35951c9f607d72f9e9a040d6fa9.